src/app/components/AnalyticsConsent/index.tsx

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable react-hooks/rules-of-hooks -- REACT_APP_ENABLE_OASIS_MATOMO_ANALYTICS can't change in runtime */
+/* eslint-disable react-hooks/rules-of-hooks -- REACT_APP_ENABLE_OASIS_MATOMO_ANALYTICS won't change in runtime */
```

Comment:
Is this line still necessary after `window.REACT_APP_ENABLE_OASIS_MATOMO_ANALYTICS` ?

Reply:
yep